hugo/content/3-queries/operators.md

```diff
@@ -212,7 +218,7 @@ For the following code samples, assume that a User has a field `scores`, which c
     ```
 
 - `{{% a_blank "$elemMatch" "https://docs.mongodb.com/manual/reference/operator/elemMatch/" %}}`
-
+    <a name="elemMatch"></a>
```

Comment:
This page talks about how to do cross references in Hugo: 
https://gohugo.io/extras/crossreferences/

You're not supposed to use pure HTML in Hugo content so probably look into that.